README.md

```diff
@@ -83,4 +83,152 @@ To make sure Docker automatically starts on boot, run
 
 `$ sudo systemctl enable docker`
 
-this shouldn't be necessary, but you may as well run it just to be safe.
\ No newline at end of file
+this shouldn't be necessary, but you may as well run it just to be safe.
+
+
+
+## JWT Tokens And Authentication
+
+> JWT is the JSON Web Token authentication plugin for the Django REST Framework.
+
+> JWT generates, access and refresh tokens for Users wanting to access information from the API
+
+>The command `permission_classes = (permissions.IsAuthenticated,)` located in `views.py` ensure that in order to access information of the API you must have logged in with your access token.
+
+•	In order to get a JWT Token, you have to register a new user. 

```

Comment:
In Mardown, bullet point lists should be created by starting the line with `* ` instead of pasting in the symbol. Spacing will be determined by Markdown so extra tabs are unnecessary.
```suggestion
* In order to get a JWT Token, you have to register a new user.
```
I also suggest this change for every item in the documentation

Reply:
Thanks for the information, Updated.